S26-documentation/12-non-breaking-space.t

```diff
@@ -1,88 +1,80 @@
 use v6;
 
-use MONKEY-SEE-NO-EVAL;
 use Test;
 
-    # non-breaking ws chars
-    my @nbchars = [
-        0x00A0, # NO-BREAK SPACE
-        0x202F, # NARROW NO-BREAK SPACE
-        0x2060, # WORD JOINER
-        0xFEFF, # ZERO WIDTH NO-BREAK SPACE
-    ];
-    # breaking ws chars
-    my @bchars = [
-        # don't use the vertical chars for this test which is single-line oriented
-        #0x000A, # LINE FEED (LF)              vertical
-        #0x000B, # LINE TABULATION             vertical
-        #0x000C, # FORM FEED (FF)              vertical
-        #0x000D, # CARRIAGE RETURN (CR)        vertical
-        #0x2028, # LINE SEPARATOR              vertical
-        #0x2029, # PARAGRAPH SEPARATOR         vertical
+use lib $?FILE.IO.parent(2).add('packages/Test-Helpers/lib');
```

Comment:
The README says to use `use lib $?FILE.IO.parent(2).add: 'packages/Test-Helpers';`. And for that to even work you would have to add the new module to the META6.json file.

Reply:
Hm, I ran a full spectest and all worked except one S17 test (I'll check and put the exact one here) that I didn't think was related, but I will add it to that META6.json (I wondered what the META6"json was for).

Reply:
It "works" because you added `/lib` to the `'packages/Test-Helpers'`. If you remove that "/lib" then it wouldn't run anymore.

Reply:
To be clear: you need to change that `use lib...` as I described before.

Reply:
Okay, I'll go back and try again. The README was added during the work on @vrurg 's branch and may not still be needed. I'll push no more changes until I think I'm really ready for your review.

Reply:
Okay, spectest ran with the same S14 failure, changes should be correct.